src/lib/components/Select.react.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+
+import React, {Component} from 'react';
+import { default as MuiSelect } from '@material-ui/core/Select';
+import lightBaseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+type Props = {
+    id: string,
+    autoFocus?: boolean,
+    autoWidth: boolean,
+    classes?: {},
+    disabled?: boolean,
+    inputRef?: () => void,
+    MenuProps?: Object,
+    multiple: boolean,
+    options?: Array<Object>,
+    /** Dash callback to update props on the server */
+    setProps?: () => void,
+    name?: string,
+    native: boolean,
+    onChange?: () => void,
+    onClose?: () => void,
+    onOpen?: () => void,
+    open?: boolean,
+    readOnly?: boolean,
+    renderValue?: Node,
+    tabIndex?: number,
+    value?: T,
```

Comment:
What is `T`?

Reply:
Honestly, I copied the props of the material-ui and updated the following examples, I missed this T (I don't know what it is). Following the [select API](https://material-ui.com/api/select/#props), the value type is any 😢.